sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeGenerator.scala

```diff
@@ -112,21 +115,21 @@ class CodeGenContext {
   }
 
   /**
-   * Returns the code to access a value in `SpecializedGetters` for a given DataType.
+   * Returns the specialized code to access a value from `inputRow` at `ordinal`.
    */
-  def getValue(getter: String, dataType: DataType, ordinal: String): String = {
+  def getValue(inputRow: String, dataType: DataType, ordinal: String): String = {
```

Comment:
well, "getter" usually means the function itself so this is better IMHO.  If anything it would be "getee" (i.e. the target of getting), but thats weird.  Other suggestions?